README.md

```diff
@@ -282,9 +284,10 @@ Table of Contents
 - [**KeePassDX**](https://github.com/Kunzisoft/KeePassDX) <sup>**[[F-Droid](https://f-droid.org/app/com.kunzisoft.keepass.libre)]**</sup>
 - [**Aegis**](https://getaegis.app/) <sup>**[[F-Droid](https://f-droid.org/app/com.beemdevelopment.aegis)]**</sup>
 - [**Bitwarden Mobile**](https://github.com/bitwarden/mobile)
-
+- [**Passwd**](https://github.com/passwdapp/passwd)
```

Comment:
The app uses a proprietary Google repository, but a quick search didn't bring up any related parts. So we can keep it I guess.

Reply:
I'm not sure about that

Reply:
Ok, so then let's include it. 👍